### PR TITLE
EN-10405: Bumping version to 8.3.99

### PR DESCRIPTION
**Pull request by TucumetaL**
Bumping version to 8.3.99 in order to have a very recognisable one that isn't a official version.

### CODE REVIEW

File: src/PIL/_version.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # Master version for Pillow
-__version__ = "8.3.2"
+__version__ = "8.3.99"
```
Comment (vilagithub) [inline src/PIL/_version.py:4]: Worth adding a comment about that version being fictional and used temporarily as a workaround required to have a pinned dependency for pip3.

**Review by vilagithub — CHANGES_REQUESTED** [review]
Voting request changes but approval granted if the comment is added.

**Review by vilagithub — APPROVED** [review]
Excellent

File: src/PIL/_version.py (hunk 1 above)
Comment (nueces) [reply src/PIL/_version.py:4]: what about using a post release version? 
```suggestion
__version__ = "8.3.2.post1"
```
https://peps.python.org/pep-0440/#public-version-identifiers

Comment (TucumetaL) [reply src/PIL/_version.py:4]: At first I thought of using a post release version, but I still think is better to have this _odd_ number to make it very clear it isn't an official release. Anyhow I'm open to change it if you think is a better choice to use a post release version.

**Pull request merged**